README.md

```diff
@@ -19,9 +19,50 @@ with `3`, e.g. `pip3` and `python3`, or follow the process for making Python
 
 ## Cloning and running it locally
 
+**Linux deployment:**
+
+Optional installation guides:
+
+Install python3 and pip3 for [linux systems](https://www.tecmint.com/install-pip-in-linux/)
+
+django installation
+`pip3 install django`
+
+Recommended for use with python 3.6 and later
+Install venv for virtual environment (optional)
+`sudo apt install -y python3-venv` - Debian
+Full python3 and venv setup [centOS](https://www.i2tutorials.com/how-to-install-python-set-up-programming-environment-on-centos/)
+
+Recommended for use with python 3.3 and 3.5 deprecated in python 3.6
```

Comment:
Since 3.8 is the latest version, we can probably just cut this section entirely out.